src/main/resources/application-development.yml

```diff
@@ -0,0 +1,20 @@
+spring:
+  config:
+    activate:
+      on-profile: development
+
+application:
+  settings:
+    serverPort: 8088
```

Comment:
Let's move this setting to the application.yml because in our case this will be the same for the development and production profiles